package/src/index.tsx

```diff
@@ -36,9 +37,10 @@ const OnramperWidget: React.FC<OnramperWidgetProps> = (props) => {
     "--primary-color": color,
     "--font-family": fontFamily,
   } as React.CSSProperties;
-  
+
   useEffect(() => {
-    props.country && i18n.changeLanguage(props.country);
+    if (props.language && isLanguageSupported(props.language))
+      i18n.changeLanguage(props.language);
```

Comment:
The initialization of the language here can be omitted, since it's happening in the API.init(). Is this covering any case or can be omitted?

Reply:
Now cleaned up. Thanks for the comment!